website/src/categories/components/combobox/code.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Combobox code
+tags: code
+APIdescription: {
+  sl-combobox: "DESCRIPTION"
```

Comment:
Can you please add the description (similar to what we have in other components will be fine)?